operational/azure/vms_without_managed_disks/README.md

```diff
@@ -0,0 +1,28 @@
+## Azure VMs Not Using Managed Disks
+
+### What it does
+This policy checks all Azure VMs and reports on any that are not using Managed Disks, which are the latest offering from Azure and are much easier to manage.
+
+### Functional Details
+When a VM that is using unmanaged disks is detected, all details of the VM are reported to the specified users.
```

Comment:
"all details"? probably just need details required to identify the VM. not sure what that equates to, but it def isn't "all details"